cmd/embedded-cluster/restore.go

```diff
@@ -933,6 +933,7 @@ func restoreCommand() *cli.Command {
 					Usage: "Disable interactive prompts.",
 					Value: false,
 				},
+				getPrivateCAsFlag(runtimeConfig),
```

Comment:
is this needed for the initial install? it will get restored from the backup.

Reply:
Can we access the backup without the private CAs? Not always

Reply:
That is what i assume so it will be needed to contact replicated app etc...